projects/storefrontlib/src/shared/components/date-picker/date-picker-formatter.service.ts

```diff
@@ -16,9 +16,9 @@ export class DatePickerFormatterService {
         .toISOString()
         .replace('.', '+')
         .replace('Z', '0');
+
       if (endOfDay) {
-        // TODO Here should be 23:59:59, but from backend I'm getting (CEST) '00:59:59', IMO backend should use UTC-0
-        date = date.replace('00:00:00', '00:59:59');
+        date = date.replace('00:00:00', '23:59:59');
```

Comment:
How is this change related to the date time picker?

Reply:
Changed when verifying if the date picker is using the correct logic or not as part of the issue related to this pr. Although not part of the final solution, this logic has been corrected here.

Reply:
It looks like an important bug fix. And it's not related to B2B epic. Then IMO we should create a separate bug-ticket for it and merge it separately to develop. It will help having a track of this bug, track of the commit fixing it and also an entry in the change log (so customers can know when it was fixed).

Reply:
This is actually only added in the b2b epic as the previous way to select start and end dates. There were issues with timezone discrepancies which is why we opted to use the datetime-picker instead.

Reply:
My bad. 👀 I thought this service was already merged to develop.